.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: TerminalImageViewer CI
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "*" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Install dependencies
+      run: sudo apt-get install -qy imagemagick
+    - name: Build
+      run: make -C src
+    - name: Install
+      run: sudo make -C src install
```

Comment:
```suggestion
```
I don't see why installing would be needed

Reply:
I added the install step to just test more of the overall logic in the repository in case someone else decides to make changes, but this step is more of the build system than the program itself. We can drop this and make the last step just use the relative path to the executable as built in the source tree. That will also ensure that we are testing what we just built and not some other copy of tiv if one gets left around.

Reply:
I think a check for new files that are added would be much better to serve that. Plus, changes to the build system are rare and probably should be manually reviewed and tested to see if there are any actual benefits.

Reply:
This step has been removed.